projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,26 @@
+<div
```

Comment:
This wrapper seems unnecessary, all of this can be handled by the host element itself.